Lasse/LittleWindows.asm

```diff
@@ -81,8 +84,7 @@ option casemap:none                             ; Preserve the case of system id
 
 ; Setting up our own stack frame
 start proc
-        mov ebp, esp
-        add esp, sp_inv_main                    ; this gives 240 bytes, more than enough for our variables
+        enter -sp_inv_main, 0                   ; this gives 240 bytes, more than enough for our variables

```

Comment:
We're now inverting the inverted stack frame size. I think it's then better to change the define for `sp_inv_main` in LittleWindows.inc to its positive counterpart (i.e. 240), rename it to `sp_main` and change this line of code accordingly. Same thing for the WndProc stack frame as well.

Reply:
That's a good idea, done.